frontend/src/app/moduls/main/main-menu/main-menu.component.sass

```diff
@@ -1,86 +1,128 @@
 .ui.visible.sidebar.inverted.vertical.menu
-    width: 13%
-    padding-left: 1.4%
+    width: 247px
+    height: 1191px
+    padding-left: 40px
```

Comment:
Are you sure this hard coded values will work for different display resolutions?

Reply:
These values ​​work correctly with the menu when zooming. I guess the other permissions will need @media. Complementing the video.
Ui with % zoom - https://youtu.be/-2nw-p_TFAE
UI with px zoom - https://youtu.be/Dwtm7GwOGxs